packages/devtools/tsconfig.json

```diff
@@ -2,14 +2,14 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "baseUrl": ".",
-    "composite": true,
     "lib": ["es6"],
     "module": "commonjs",
-    "noImplicitThis": true,
     "outDir": "lib",
-    "strictNullChecks": true,
     "target": "es2015",
     "types": ["node", "jest"],
+
+    /* The way setupGanache is written is hard to re-write in 5 min to fix this */
```

Comment:
Curious to hear a quick summary of what makes it hard?

Reply:
It just took longer than a couple minutes and I didn't want to start editing code. I think the quick summary is that it returns one of two objects which do not overlap with each other and behave quite differently.